### DISCUSSION TITLE
The program has been very unstable recently.

### DISCUSSION THREAD

**Discussion by 2p990i9hpral** [GaiZhenbiao/ChuanhuChatGPT#976] [Q&A]
The program has been very unstable recently.

Recently (probably a few months ago), there has been a very frequent problem where there is no response when entering a message or the problem is interrupted while creating a response.
Is this only happening to me? Or is there actually some kind of problem?

This is the error message. 
"Exception ignored in: <generator object OpenAIClient._decode_chat_response at 0x000001BDAD0AF840>
Traceback (most recent call last):
   File "D:\ChuanhuChatGPT-main\ChuanhuChat\lib\site-packages\anyio\_backends\_asyncio.py", line 802, in run"

**Comment by GaiZhenbiao** [comment]
Maybe this is a package version issue. It shouldn't behave different if you didn't make any changes though. You can try pulling the latest code
```
git pull
```
Then upgrade all requirements to the current supported version
```
pip install -r requirements.txt --upgrade
```
